storage/src/backend/registry.rs

```diff
@@ -37,6 +39,7 @@ const REDIRECTED_STATUS_CODE: [StatusCode; 2] = [
 ];
 
 const REGISTRY_DEFAULT_TOKEN_EXPIRATION: u64 = 10 * 60; // in seconds
+const MAX_RETRIES: u32 = 1;
```

Comment:
It is unnecessary to add the retry mechanism because the upper caller will retry if any error happens.